cves/kernel/CVE-2020-11668.yml

```diff
@@ -460,7 +464,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    Unfortunately, this vulnerability is a result of missing key requirements. This vulnerability could be constructed with a USB device that did not have valid metadata. This then causes the dereferencing of a NULL-pointer, which could then cause a denial of service as the research suggested.
```

Comment:
Multiple lines. Could explain more about the mistake rather than describe the vulnerability